elastic/kibana
# [Fleet] Trim whitespace from package policy names

## Summary

Closes #122015

when a package policy is created or updated with whitespace at the start or end of the name, we now remove this before saving. 

**Tests**
I've added an automated test for both create and update.

1. create a package policy with whitespace at the start or end, notice the whitespace is not included after creation
2. update a package policy to add whitespace to the name, notice the whitespace is not saved.
